### PR TITLE
Improve external volumes examples

### PR DESCRIPTION
**Pull request by sascala**
## Description
Have the external volumes examples actually employ volumes
https://jira.mesosphere.com/browse/DOCS-1505

## Urgency
- [X] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/storage/external-storage.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ You can specify an external volume in your Marathon app definition. [Learn more
       "instances": 1,
       "cpus": 0.1,
       "mem": 32,
-      "cmd": "/usr/bin/tail -f /dev/null",
+      "cmd": "mkdir my-test-vol && touch my-test-vol/test.txt && ls my-test-vol/",
```
Comment (gpaul) [inline 1.9/storage/external-storage.md]: I believe this should be 
```
mkdir test-rexray-volume/test-dir && touch test-rexray-volume/test-dir/test.txt && ls test-rexray-volume/test-dir/
```

The `my-test-vol` property that is specified for this volume is the name of the volume in eg., your AWS account, and is unrelated to the directory path in the container. The directory path in the container is given by the `containerPath` property, which in this case is `test-rexray-volume`.

File: 1.9/storage/external-storage.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ Below is a sample app definition that uses a Docker container and specifies an e
...
```
Comment (gpaul) [inline 1.9/storage/external-storage.md]: I believe this should be 
```
mkdir /data/test-rexray-volume/test-dir && touch /data/test-rexray-volume/test-dir/test.txt && ls /data/test-rexray-volume/test-dir/
```

The directory path in the container is given by the `containerPath` property, which in this case is `/data/test-rexray-volume`. Note that it is an absolute path in this case, as this is using the Docker containerizer, which expects absolute paths for `containerPath` and mounts the external volume at exactly the given path in the container.

**Review by gpaul — CHANGES_REQUESTED** [review]
I noticed that the commands are wrong.

If you'd like to test that it is correct now and was wrong before, spin up a cluster on CCM. 

Next, use the following CMD instead.

```
"cmd": "date >> test-rexray-volume/test.txt; cat test-rexray-volume/test.txt",
```
(It's important to use the '>>' or "append" operator for this test.)

If the external volume is being used correctly, the logs of successive runs of the application will show more and more lines as part of the `cat` operation.

Finally, submit the app definition and wait for it to run - it should run several times and every time the list of dates printed to the stdout log of the instances should grow longer. If it doesn't it means that the `test.txt` file is not being persisted between runs and the config is wrong.

The same test holds for the docker containerizer, but the paths should be `/data/test-rexray-volume/test.txt` instead.

**Comment by sascala** [comment]
Thank you for the suggested command, @gpaul . I've changed the command and added explanatory text.

**Review by gpaul — APPROVED** [review]
LGTM :+1:

**Pull request merged**